python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -229,152 +292,188 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
-        madDists = sigmaMad(dists)
-        meanDists = np.mean(dists)
-
-        rmsDists = np.sqrt(np.mean(np.array(dists)**2))
-        axHist.set_xlim(meanDists - 2.0*rmsDists, meanDists + 2.0*rmsDists)
-        lineMedian = axHist.axvline(medDists, color="k", label="Median: {:0.3f}".format(medDists))
-        lineMad = axHist.axvline(medDists + madDists, color="k", ls="--",
-                                 label="sigma MAD: {:0.3f}".format(madDists))
-        axHist.axvline(medDists - madDists, color="k", ls="--")
-        lineMean = axHist.axvline(meanDists, color="C0", label="Mean: {:0.3f}".format(meanDists))
-        lineRms = axHist.axvline(meanDists + rmsDists, color="C0", ls="--",
-                                 label="RMS: {:0.3f}".format(rmsDists))
-        axHist.axvline(meanDists - rmsDists, color="C0", ls="--")
-
-        linesForLegend = [lineHW, lineInit, lineRefit, fitScatter, lineMedian, lineMad, lineMean, lineRms]
-        fig.legend(handles=linesForLegend, fontsize=8, bbox_to_anchor=(1.0, 0.99),
-                   bbox_transform=fig.transFigure, ncol=2)
-
-        axHist.hist(dists, bins=100, histtype="step", label="Refit", color="C0")
-        axHist.hist(distsHW, bins=100, histtype="step", label="HW", color="C0", alpha=0.5)
+        axHist.set_ylabel("Number", fontsize=8)
+        axHist.set_xlabel("Distance to Line Fit ({})".format(statsUnitStr), fontsize=8)
+        axHist.tick_params(labelsize=8)
+        nSigToPlot = 3.5
+        axHist.set_xlim(meanDistsScaled - nSigToPlot*madDistsScaled,
+                        meanDistsScaled + nSigToPlot*madDistsScaled)
+        lineMedian = axHist.axvline(medDistsScaled, color="k", label="Median: {:0.2f} {}".
+                                    format(medDistsScaled, statsUnitStr))
+        lineMad = axHist.axvline(medDistsScaled + madDistsScaled, color="k", ls="--",
+                                 label=r"$\sigma_{MAD}$" + ": {:0.2f} {}"
+                                 .format(madDistsScaled, statsUnitStr))
+        axHist.axvline(medDistsScaled - madDistsScaled, color="k", ls="--")
+        lineRms = axHist.axvline(meanDistsScaled + rmsDistsScaled, color="C0", ls="--",
+                                 label="RMS: {:0.2f} {}".format(rmsDistsScaled, statsUnitStr))
+        axHist.axvline(meanDistsScaled - rmsDistsScaled, color="C0", ls="--")
+
+        linesForLegend = [lineMedian, lineMad, lineRms]
+        fig.legend(handles=linesForLegend, fontsize=7, loc="lower right", bbox_to_anchor=(0.96, 0.84),
+                   bbox_transform=fig.transFigure, ncol=1)
+
+        axHist.hist(distsScaled, bins=200, histtype="step", label="ODR Fit", color="C0")
+        axHist.hist(distsScaledHW, bins=200, histtype="step", label="HW", color="C0", alpha=0.5)
 
         alphas = [1.0, 0.5]
         handles = [Rectangle((0, 0), 1, 1, color="none", ec="C0", alpha=a) for a in alphas]
-        labels = ["Refit", "HW"]
+        labels = ["ODR Fit", "HW"]
```

Comment:
This should distinguish between the first round of fitting and the second.

Reply:
How so?  I re-named `Refit` to `ODR Fit` because the former was never really distinguishing the difference between the first and second rounds of the fit (I was thinking we are only really distinguishing between the "Hard Wired" fit and our final fit on the plots.  Suggestions other than `ODR Fit` welcome (e.g. `Final Fit` or just `Fit`)?

Reply:
Actually, there is nothing being fit "hard wired" wired case.  It is meant to be a sort of "average/reference" fit so that one can identify any field-to-field variations in the location of the stellar locus (particularly in the "P1" principal component).  We compute the mean and standard deviation from the hardwired fit coefficients; a shift in the former would indicate something is "different" in this field (a tract in the current usage), whether it be astrophysical due to a very different stellar population from the one used for the hardwired fit or something amiss with the data reduction.  In the "fit" versions, the mean is ~zero by definition, so this information is lost (it is only the `sigmaMad` of the fit distribution that is the "metric" we typically look at).

I'm open to making a `"hardwired" -> "fixed"` name change, but wouldn't mind punting that to the ticket where this all gets implemented in `analysis_tools` (which did inherit the "hardwired" terminology).  Is that reasonable?

Reply:
Yup, I'm happy to defer any changes to the analysis_tools port.